### PR TITLE
Add contents to lockfile for lockfile validation

### PR DESCRIPTION
**Pull request by ChengTian**
Counterpart of https://github.com/aspnet/XRE/pull/1285
@davidfowl @anurse

### CODE REVIEW

File: src/NuGet.ProjectModel/LockFileFrameworkInformation.cs [hunk 1]
```diff
@@ -0,0 +1,20 @@
+﻿// Copyright (c) Microsoft Open Technologies, Inc. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+using System.Collections.Generic;
+
+namespace NuGet.ProjectModel
+{
+    public class LockFileFrameworkInformation
```
Comment (davidfowl) [inline src/NuGet.ProjectModel/LockFileFrameworkInformation.cs]: Not sure about this name

Comment (ChengTian) [inline src/NuGet.ProjectModel/LockFileFrameworkInformation.cs]: LockFileDependencyGroup?

Comment (ChengTian) [inline src/NuGet.ProjectModel/LockFileFrameworkInformation.cs]: Fixed

**Comment by ChengTian** [comment]
Made code review changes

**Comment by davidfowl** [comment]
:shipit:

**Pull request merged**